### ISSUE TITLE
[FAST-54] [Firefox] Color and contrast of text over the image not match with WACG standards

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3074]
**Steps to Reproduce**
1) Go to the https://www.topcoder.com/community/accessibility-testing page
2) Enter into inspect element and go to the tab of Accessibility
3) Observe the color contrast over the image not good

**Expected Result**
The color contrast between background and foreground content should be good contrast.

**Actual Result**
The color contrast between background and foreground content (i.e. usually text) should be great enough to ensure legibility

**Device**: Desktop/Labtop
**Browser**: Firefox

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: 
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=2cd27847%2Dc944%2D4866%2Da87f%2D5eec4e505c52&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fkmurti%2FBug3

**Comment by Dara-K** [comment]
This is more about the contrast between image and text which depends on the image largely.

Here is a darker image to use for this page
![Banner accessibility](https://user-images.githubusercontent.com/25185079/61913843-0ef53300-af47-11e9-820b-bc8a0bbe4780.png)

**Comment by kkartunov** [comment]
Done.
